### PR TITLE
fix(components): focus indicator when textarea is resized (VIV-905)

### PR DESCRIPTION
**Pull request by olaf-k**

### CODE REVIEW

**Review by rachelbt — CHANGES_REQUESTED** [review]
I think that we need to keep the focus design also here and no the native one.

**Comment by olaf-k** [comment]
> I think that we need to keep the focus design also here and no the native one.

the difference is subtle but it's here indeed 👍 I'll have a look

File: libs/components/src/lib/text-area/text-area.scss [hunk 1]
```diff
@@ -26,9 +29,9 @@ $low-ink-color: --_low-ink-color;
 	@include connotation.connotation;
 
 	display: inline-grid;
-	width: 100%;
-	gap: 4px;
+	width: inherit;
 	grid-template-columns: 1fr max-content;
```
Comment (rachelbt) [inline libs/components/src/lib/text-area/text-area.scss:33]:
```suggestion
	grid-template-columns: 1fr;
```
not relevant to this. PT but there are no case where there are 2 columns in the grid

File: libs/components/src/lib/text-area/text-area.scss [hunk 2]
```diff
@@ -88,19 +82,8 @@ $low-ink-color: --_low-ink-color;
...
-}
-
-.focus-indicator {
-	#{focus-variables.$focus-stroke-gap-color}: transparent;
-
-	pointer-events: none;
-
-	.fieldset:not(:focus-visible, :focus-within) > & {
-		display: none;
+	&:focus {
```
Comment (rachelbt) [inline libs/components/src/lib/text-area/text-area.scss:85]: should we add TODO: remove once VIV-904 is done?

**Review by rachelbt — APPROVED** [review]
LGTM

**Pull request merged**